omod/src/main/webapp/pages/results.gsp

```diff
@@ -56,7 +97,11 @@
     </thead>
     <tbody>
     <% results.each { %>
+    <script>
+        patients.push("${ui.encodeHtmlContent(it.name)}");
+    </script>
     <tr>
+        <td style="display: none">rule_${it.rule.ruleId}</td>
         <td>${ui.encodeHtmlContent(it.rule.ruleName ?: '')}</td>
         <td>${it.patient.patientId ?: it.patientProgram.patientProgramId}</td>
```

Comment:
@shivtej1505 Instead of the patientId, why don't we need to find a way of showing the preferred patient identifier